src/parser/jobs/sch/modules/ScholarCooldownDowntime.tsx

```diff
@@ -0,0 +1,23 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React, {Fragment} from 'react'
+
+export default class ScholarCooldownDowntime extends CooldownDowntime {
+	override trackedCds = [
+		{
+			cooldowns: [this.data.actions.CHAIN_STRATAGEM],
+			allowedAverageDowntime: 7500,
+			firstUseOffset: 10000,
```

Comment:
These don't seem to reflect actual usage in openers. The allowed average here also seems really high, although you don't actually need to configure it unless there's a good reason for an explicit setting.

Reply:
I left it loose because I wasn't sure why Chain Strat was left loose. I would think maybe 1 or 2 GCDs would be more appropriate. But if leaving it unset works that's probably what I'll do then since adding it is just to make sure you're casting it every minute.

Reply:
Average allowed downtime defaults to 1250ms, but note that this is the average, so you can have up to double that. With 7500, you're saying they can effectively leave it off cooldown for 15 seconds. I'm seeing Aetherflow used pretty consistently beyond 10s into the fight so this first use offset seems wrong. Chain seems to be around 5-6 seconds (3rd GCD?), so I think that might need tightening too, but compare openers across regions since they tend to differ, especially JP/KR.

Reply:
Ah, I didn't realize that the grace period effectively doubled. Yeah that would be way to long. I'll take a close look at openers tomorrow to make sure these values get set correctly.

Reply:
After looking a some guides and some top logs it seems like Chain Strat always gets casted right before the 4th GCD (to match up buff windows), which at max GCD is 10s. So 10s for chain strat first use seems fine. The first Aetherflow cast should generally be casted before the first Chain Strat so that you can fit 5 energy drains in the Chain Strat window. So I picked 7.5s for that since Aetherflow would need to be casted before the 3rd GCD.

I think the averageAllowedDowntime should be fine at the default since it's just used to calculate the max number of casts, and the only time I saw Chain Strat being delayed looked like a specific raid strat.

Reply:
I'd maybe check some logs from other regions to make sure there's no different approach used in like, JP since they tend to have diff openers that are equally viable. Also worth checking some non-top logs, since the higher ranked players tend to have approaches highly optimised that don't reflect the general case. Usually 80-90th percentile is a good baseline for generalising.

Reply:
I checked a few more logs in the range you suggested, on both savage and normal, and in multiple regions. The opener seems to hold pretty steady. The variations that moved Chain Strat around only seemed to move it forwards, so those variations would not be penalized by the current setting.
1. Precast Broil
2. Biolysis
  a. Weave Aetherflow
3. Broil
  a. Optional Energy Drain Weave (you can fit a max of 5 Energy Drain casts in Chain Strat duration, so you do one early or one late)
4. Broil
  a. Weave Chain Strat
5. First burst window (5x Broil, 1x Dissipation, 5x Energy Drain) with slight variations

Reply:
Awesome, thanks for triple checking!